plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -836,8 +835,18 @@ def modify_xml(self):
         self._templatize_pkgid(other)
         self._templatize_pkgid(filelists)
 
+        text_other = utils.element_to_text(other)
+
+        other_l = text_other.find(">")
+        other_r = text_other.rfind("</")
```

Comment:
iiuc, the intention here is to put all the changelog elements into one verbatim/endverbatim templatetag. My suggestion is to search for the first opening changelog tag and for the last closing changelog tag:

    other_l = text_other.find("<changelog")
    other_r = text_other.rfind("</changelog>")